package.json

```
@@ -57,6 +57,7 @@
     "ini": "3.0.1",
     "isomorphic-git": "1.21.0",
     "livereload-js": "3.4.1",
+    "minimatch": "^6.1.6",
```

Is this a net new dependency or just a transitive dependency that we are turning into first level?

We also pin all deps by default:

```suggestion
    "minimatch": "6.1.6",
```

---

Its a new dependency I added for glob matching.